<launch>
  <arg name="robot_namespace" default="bvr_SIM"/>
  <arg name="arm_namespace" default="main_arm_SIM"/>

  <node name="arm_control" pkg="arm_control" type="arm_control_node" output="screen">
    <param name="robot_namespace" value="$(arg robot_namespace)"/>
    <param name="arm_namespace" value="$(arg arm_namespace)"/>
  </node>
</launch>
